app/assets/stylesheets/components/curriculum_details.scss

```diff
@@ -51,10 +50,13 @@
 
       h3 {
         margin-bottom: .2em;
-      }
-
-      p {
-        font-size: .9em;
+        margin-top: 20px;
+        font-size: 18px;
+        font-weight: bold;
+        letter-spacing: 0.2px;
+        font-family: OpenSans;
+        color: #4a4a4a;
+        color: var(--greyish-brown);
```

Comment:
Good catch @sdg123, --greyish-brown is `#4a4a4a` so only one color value is needed here. We actually have a color variable for `#4a4a4a` its `$text-primary`. Can you use that instead here please.